{"cells":[{"source":"<a href=\"https://www.kaggle.com/code/nghiemdinhnam/vffinder-retrain?scriptVersionId=173123719\" target=\"_blank\"><img align=\"left\" alt=\"Kaggle\" title=\"Open in Kaggle\" src=\"https://kaggle.com/static/images/open-in-kaggle.svg\"></a>","metadata":{},"cell_type":"markdown"},{"cell_type":"code","execution_count":1,"id":"c980350b","metadata":{"execution":{"iopub.execute_input":"2024-04-21T06:54:24.28423Z","iopub.status.busy":"2024-04-21T06:54:24.283892Z","iopub.status.idle":"2024-04-21T06:55:09.387309Z","shell.execute_reply":"2024-04-21T06:55:09.386068Z"},"papermill":{"duration":45.109843,"end_time":"2024-04-21T06:55:09.389766","exception":false,"start_time":"2024-04-21T06:54:24.279923","status":"completed"},"tags":[]},"outputs":[{"name":"stdout","output_type":"stream","text":["  \u001b[1;31merror\u001b[0m: \u001b[1msubprocess-exited-with-error\u001b[0m\r\n","  \r\n","  \u001b[31m×\u001b[0m \u001b[32mgit clone --\u001b[0m\u001b[32mfilter\u001b[0m\u001b[32m=\u001b[0m\u001b[32mblob\u001b[0m\u001b[32m:none --quiet \u001b[0m\u001b[4;32mhttps://github.com/pyg-team/pytorch_geometric.git\u001b[0m\u001b[32m \u001b[0m\u001b[32m/tmp/\u001b[0m\u001b[32mpip-req-build-sboy01xv\u001b[0m did not run successfully.\r\n","  \u001b[31m│\u001b[0m exit code: \u001b[1;36m128\u001b[0m\r\n","  \u001b[31m╰─>\u001b[0m \u001b[31m[1 lines of output]\u001b[0m\r\n","  \u001b[31m   \u001b[0m fatal: unable to access 'https://github.com/pyg-team/pytorch_geometric.git/': Could not resolve host: github.com\r\n","  \u001b[31m   \u001b[0m \u001b[31m[end of output]\u001b[0m\r\n","  \r\n","  \u001b[1;35mnote\u001b[0m: This error originates from a subprocess, and is likely not a problem with pip.\r\n","\u001b[1;31merror\u001b[0m: \u001b[1msubprocess-exited-with-error\u001b[0m\r\n","\r\n","\u001b[31m×\u001b[0m \u001b[32mgit clone --\u001b[0m\u001b[32mfilter\u001b[0m\u001b[32m=\u001b[0m\u001b[32mblob\u001b[0m\u001b[32m:none --quiet \u001b[0m\u001b[4;32mhttps://github.com/pyg-team/pytorch_geometric.git\u001b[0m\u001b[32m \u001b[0m\u001b[32m/tmp/\u001b[0m\u001b[32mpip-req-build-sboy01xv\u001b[0m did not run successfully.\r\n","\u001b[31m│\u001b[0m exit code: \u001b[1;36m128\u001b[0m\r\n","\u001b[31m╰─>\u001b[0m See above for output.\r\n","\r\n","\u001b[1;35mnote\u001b[0m: This error originates from a subprocess, and is likely not a problem with pip.\r\n"]}],"source":["!git config --global --unset https.proxy\n","!pip install -q git+https://github.com/pyg-team/pytorch_geometric.git\n","!mkdir Model\n","!mkdir -p Data/result"]},{"cell_type":"code","execution_count":2,"id":"f8ac13fd","metadata":{"execution":{"iopub.execute_input":"2024-04-21T06:55:09.396088Z","iopub.status.busy":"2024-04-21T06:55:09.395524Z","iopub.status.idle":"2024-04-21T06:55:30.367946Z","shell.execute_reply":"2024-04-21T06:55:30.36705Z"},"papermill":{"duration":20.977818,"end_time":"2024-04-21T06:55:30.370047","exception":false,"start_time":"2024-04-21T06:55:09.392229","status":"completed"},"tags":[]},"outputs":[{"name":"stdout","output_type":"stream","text":["Cloning into 'vffinder_retrain'...\r\n","fatal: unable to access 'https://github.com/DinhNam99/vffinder_retrain.git/': Could not resolve host: github.com\r\n"]}],"source":["!git clone https://github.com/DinhNam99/vffinder_retrain.git"]},{"cell_type":"code","execution_count":3,"id":"b0fc2516","metadata":{"execution":{"iopub.execute_input":"2024-04-21T06:55:30.37612Z","iopub.status.busy":"2024-04-21T06:55:30.375826Z","iopub.status.idle":"2024-04-21T06:55:31.410511Z","shell.execute_reply":"2024-04-21T06:55:31.4094Z"},"papermill":{"duration":1.040306,"end_time":"2024-04-21T06:55:31.412826","exception":false,"start_time":"2024-04-21T06:55:30.37252","status":"completed"},"tags":[]},"outputs":[{"name":"stdout","output_type":"stream","text":["python: can't open file '/kaggle/working/vffinder_retrain/Main_VULJIT_Detection.py': [Errno 2] No such file or directory\r\n"]}],"source":["!python vffinder_retrain/Main_VULJIT_Detection.py \\\n","--graph_dir='/kaggle/input/embedding-ast-silent/embedding_ast'  \\\n","--train_file='/kaggle/input/data-slient-fix/cross_train.txt' \\\n","--test_file='/kaggle/input/data-slient-fix/cross_test.txt' \\\n","--model_dir='/kaggle/working/Model' \\\n","--max_epochs=50 \\\n","--model_name='gat' \\\n","--hidden_size=32 \\\n","--GNN_type=\"GAT\"\n"]}],"metadata":{"kaggle":{"accelerator":"gpu","dataSources":[{"datasetId":3486548,"sourceId":6090932,"sourceType":"datasetVersion"},{"datasetId":3486547,"sourceId":6093388,"sourceType":"datasetVersion"},{"datasetId":3491582,"sourceId":6096424,"sourceType":"datasetVersion"},{"datasetId":3492589,"sourceId":6097804,"sourceType":"datasetVersion"},{"datasetId":3663181,"sourceId":6359419,"sourceType":"datasetVersion"},{"datasetId":3495576,"sourceId":6359436,"sourceType":"datasetVersion"},{"datasetId":3486602,"sourceId":6365199,"sourceType":"datasetVersion"}],"dockerImageVersionId":30497,"isGpuEnabled":true,"isInternetEnabled":false,"language":"python","sourceType":"notebook"},"kernelspec":{"display_name":"Python 3","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.10.10"},"papermill":{"default_parameters":{},"duration":78.228947,"end_time":"2024-04-21T06:55:32.134663","environment_variables":{},"exception":null,"input_path":"__notebook__.ipynb","output_path":"__notebook__.ipynb","parameters":{},"start_time":"2024-04-21T06:54:13.905716","version":"2.4.0"}},"nbformat":4,"nbformat_minor":5}